tests/Doctrine/Tests_PHP74/Common/Reflection/Dummies/TokenParserNullValueTestClass.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Doctrine\Tests_PHP74\Common\Reflection\Dummies;
+
+trait TokenParserNullValueTestClass
```

Comment:
Am I correct assuming this is failing due to `static function ($value)` having no name specified? If so then please rename the trait and test to make it obvious what's "wrong"

Reply:
I changed the name, do you think it's better like this?

Reply:
Way better :+1: